### PR TITLE
Update standard names for tropopause_find

### PR DESCRIPTION
**Pull request by jimmielin**
Fixes https://github.com/ESCOMP/CAM-SIMA/issues/308
Requires a companion PR in CAM-SIMA to update the standard name of `tropp_slices` there as well: https://github.com/ESCOMP/CAM-SIMA/pull/329

### CODE REVIEW

File: schemes/tropopause_find/tropopause_find.meta [hunk 1]
```diff
@@ -115,13 +115,13 @@
   standard_name = tropopause_air_pressure_from_climatology_dataset
   units = Pa
   type = real | kind = kind_phys
-  dimensions = (horizontal_dimension, number_of_months_in_year)
+  dimensions = (horizontal_dimension, number_of_time_slices_in_climatology_dataset)
```
Comment (nusbaume) [inline schemes/tropopause_find/tropopause_find.meta:118]: I realize we should probably be as explicit as possible and label this a `tropopause_climatology_dataset`:

```suggestion
  dimensions = (horizontal_dimension, number_of_time_slices_in_tropopause_climatology_dataset)
```

The same request holds below and for the equivalent uses  in CAM-SIMA as well.  Thanks!

**Review by nusbaume — CHANGES_REQUESTED** [review]
Looks good, but I did have one request for the new standard name.

File: schemes/tropopause_find/tropopause_find.meta (hunk 1 above)
Comment (jimmielin) [reply schemes/tropopause_find/tropopause_find.meta:118]: Thanks @nusbaume! I updated throughout to clarify the standard names for the tropopause climatology dataset related variables.

* Updated `tropopause_air_pressure_from_climatology_dataset` -> `tropopause_air_pressure_from_tropopause_climatology_dataset`
* Updated `number_of_time_slices_in_climatology_dataset` -> `number_of_time_slices_in_tropopause_climatology_dataset`
* Updated `tropopause_calendar_days_from_climatology` -> `tropopause_calendar_days_from_tropopause_climatology`

**Review by PeterHjortLauritzen — APPROVED** [review]

**Review by nusbaume — APPROVED** [review]
Looks great now, thanks @jimmielin!

**Review by peverwhee — APPROVED** [review]

**Pull request merged**